sale_timesheet_invoice_description/tests/test_sale_timesheet_description.py

```diff
@@ -61,28 +61,26 @@ def setUpClass(cls):
     def _test_sale_time_description(self, desc_option, expected):
         self.sale_order.timesheet_invoice_description = desc_option
         self.sale_order.action_confirm()
-        invoice_id = self.sale_order.with_context(
+        invoice = self.sale_order.with_context(
             test_timesheet_description=True
-        ).action_invoice_create()
-        invoice = self.env["account.invoice"].browse(invoice_id)
-        self.assertEqual(invoice.invoice_line_ids[0].name, expected)
+        )._create_invoices()
+        self.assertEqual(invoice[0].line_ids[0].name, expected)
```

Comment:
I think `invoice_line_ids` (or similar) still exists in v13, but this one seems the general items one2many. And why `invoice[0]`? Isn't there only one?

Reply:
Travis tells me an error without the [0]

Reply:
I suppose you have deleted your comment. Please read carefully what I'm saying.

Reply:
For reasons of future compatibility, it will be better to use "line_ids"

invoice[0] change it to avoid a Travis error

Reply:
Which future compatibility? this is already 13.0 in which the merge has been done. And what is the error before putting `invoice[0]`?